composer/loggers/progress_bar_logger.py

```diff
@@ -189,7 +189,19 @@ def log_data(self, state: State, log_level: LogLevel, data: Dict[str, Any]) -> N
         # log to console
         if self.should_log(state, log_level):
             data_str = format_log_data_value(data)
-            log_str = f'[{log_level.name}][batch={int(state.timestamp.batch)}]: {data_str}'
+            assert state.max_duration is not None, 'max_duration should be set'
```

Comment:
If something is logged on `Event.INIT`, then max duration could be none, so probs best not to prepend it?